File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Weathers/Forecast.java

package com.appsbygu.kadai_weather_java.models.apis.Weathers;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

public class Forecast extends RealmObject {

    @SerializedName("dateLabel")
    @Expose
    private String dateLabel;
    @SerializedName("telop")
    @Expose
    private String telop;
    @SerializedName("date")
    @Expose
    private String date;
    @SerializedName("image")
    @Expose
    private Image image;

    public String getDateLabel() {
        return dateLabel;
    }

    public String getTelop() {
        return telop;
    }

    public String getDate() {
        return date;
    }

    public Image getImage() {
        return image;
    }

}

File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/weathers/Forecast.kt
package com.appsbygu.kadai_weather_java.models.apis.weathers

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import io.realm.RealmObject

open class Forecast : RealmObject() {

    @SerializedName("dateLabel")
    @Expose
    var dateLabel: String? = null
    @SerializedName("telop")
    @Expose
    var telop: String? = null
    @SerializedName("date")
    @Expose
    var date: String? = null
    @SerializedName("image")
    @Expose
    var image: Image? = null

}
